website/addons/base/views.py

```diff
@@ -202,7 +204,27 @@ def create_waterbutler_log(payload, **kwargs):
     auth = Auth(user=user)
     node = kwargs['node'] or kwargs['project']
 
-    if action in (NodeLog.FILE_MOVED, NodeLog.FILE_COPIED):
+    try:
+        dest = payload['destination']
```

Comment:
This logic should only be run if there is a move performed.
Move this below the  `if action in (NodeLog.FILE_MOVED, NodeLog.FILE_COPIED, NodeLog.FILE_RENAMED):` line to avoid the unnecessary error handling.